### DISCUSSION TITLE
LPV Control

### DISCUSSION THREAD

**Discussion by rahimme2k7** [yalmip/YALMIP#1226] [General discussion]
Dear Altruist,
I am a new research student in control systems and planning to research LPV Polytopic based Autonomous Vehicle Driving  Control Systems using MATLAB-YALMIP. It would be very helpful for me if anyone could share some sample codes related to LPV Polytopic or reduced Polytopic or LFT or Grid-based. Surely, I will acknowledge your help.

**Comment by johanlofberg** [comment]
https://yalmip.github.io/example/lpvstatefeedback/
